projects/demo/src/app/overlay-clipping/overlay-clipping.demo.html

```diff
@@ -0,0 +1,203 @@
+      magna. Phasellus tristique posuere dui, ut tempus felis sagittis quis. Integer iaculis ultrices elit, sed
+      venenatis eros. Vivamus interdum semper velit eget gravida. Sed finibus eget lacus sed semper. Suspendisse
+      fringilla, tellus in molestie cursus, sapien purus volutpat lacus, eget venenatis erat est vitae libero. Aliquam
+      et orci hendrerit, consequat purus non, imperdiet ipsum.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eu odio nisi. Vestibulum dignissim eget massa sit
+      amet feugiat. Quisque auctor mattis quam eu suscipit. Morbi ipsum risus, feugiat vitae sem at, tincidunt elementum
+      magna. Phasellus tristique posuere dui, ut tempus felis sagittis quis. Integer iaculis ultrices elit, sed
+      venenatis eros. Vivamus interdum semper velit eget gravida. Sed finibus eget lacus sed semper. Suspendisse
+      fringilla, tellus in molestie cursus, sapien purus volutpat lacus, eget venenatis erat est vitae libero. Aliquam
+      et orci hendrerit, consequat purus non, imperdiet ipsum.
+    </p>
+  </clr-wizard-page>
+
+  <clr-wizard-page>
+    <ng-template clrPageTitle>Title for page 2</ng-template>
+    <!-- mandatory -->
+    <ng-template clrPageNavTitle>Step 2</ng-template>
+    <!-- optional -->
+    <p class="content-for-page-2">Content for step 2</p>
+  </clr-wizard-page>
+</clr-wizard>
+
+<ng-template #signpostDemo>
+  <clr-signpost>
+    <clr-signpost-content *clrIfOpen>
+      <h3>Default Signpost</h3>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+      Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </clr-signpost-content>
+  </clr-signpost>
+  <!-- <a href="javascript://" role="tooltip" aria-haspopup="true" class="tooltip tooltip-lg">
+    <cds-icon status="info" shape="folder" size="24"></cds-icon>
+    <span class="tooltip-content">Lorem ipsum dolor sit amet, consectetur adipisicing elit</span>
+  </a> -->
+  <clr-tooltip>
+    <cds-icon
+      status="info"
+      title="Tooltip icon"
+      clrTooltipTrigger
+      shape="info-standard"
+      size="18"
+      aria-label="blank id string"
+    ></cds-icon>
+    <clr-tooltip-content [id]="blankStringId" clrPosition="top-right" clrSize="md">
+      <span
+        >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.</span
+      >
```

Comment:
These mangled `span` tags should be fixed.

```html
<span>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
  consequat.
</span>
```

Reply:
Hmm, I don't like it either. The commit hook refused to accept this without the _lint:format:fix, which automatically mangled it the way you see it.
The _lint:format:fix should not be generating this style of markup at all, I think

Reply:
It's a quirk of `prettier`. We just have to work around it.

Reply:
It's a quirk of `prettier` because it think the lack of whitespace is significant. We can work around it though by adding the whitespace.

Reply:
Thanks!